devcareer/code-jammers-backend
# #174626782 Create tourist center and state models

### What does this PR do?

This PR creates
- a state model
- a tourist center model
- state migrations and seeders
- tourist center migrations and seeders

### Description of tasks to be completed?

- create state model
- create tourist center model
- create state migrations
- create tourist centeer migrations
- create state seeders
- create tourist center seeders
- install sequelize
- install sequelize-cli
   
### How should this be manually tested/checked?
Clone the PR and checkout to the branch ft-tourist-centers-cities-models-174626782
- run ```npm i``` to install the dependencies
- create a database and add the connection string to your .env file
  eg DEV_DATABASE_URL = postgres://postgres:DATABASE_PASSWORD@localhost:5432/DATABASE_NAME 
- run ```npm test``` all tests should pass
- run ```npm migrate```  to create the required database tables
- run ```npm seed``` to seed the database
    
### What are the relevant pivotal tracker stories?
[#174626782 ](https://www.pivotaltracker.com/n/projects/2463429/stories/174626782)
